README.md

```diff
@@ -139,19 +76,29 @@ Example command:
 
 Clone this repository to your desired folder:
 
+- Click on the green button on the top right corner with the text code
+- Click on SSH
+- Copy the text that appears below
+- Type "git clone" in your terminal and past the copied text next to "git clone" and press enter
 <!--
 Example commands:
 
 ```sh
   cd my-folder
   git clone git@github.com:myaccount/my-project.git
```

Comment:
[OPTIONAL]

- [ ] It would be nicer to give the link of the project to be cloned:
```sh
git clone git@github.com:Nejjari001/MyEnumerable.git
```
- [ ] As well as how to run the project locally like
```ruby
`ruby -r "./my_list.rb" -e "p MyList.new(1,2,4)"`

```